README.md

```diff
@@ -42,46 +42,168 @@ These elements are described in different folders. All the modelling happens wit
 
 The files that are outside from the `openfisca_country_template` folder are used to set up the development environment.
 
+## Packaging your Country Package for Distribution
 
-## Installing
+Country packages are python distributions. To distribute your package via `pip install`, follow the steps given by the [Python Packaging Authority](https://python-packaging-user-guide.readthedocs.io/distributing/).
```

Comment:
via `pip`. 
`pip` is the package manager, `pip install` the command, and we distribute via `pip` ;).